search-commons/src/test/java/no/unit/nva/search/ticket/TicketClientTestV2.java

```diff
@@ -0,0 +1,180 @@
+package no.unit.nva.search.ticket;
+
+import static no.unit.nva.common.Containers.container;
+import static no.unit.nva.common.Containers.indexingClient;
+import static no.unit.nva.constants.IndexMappingsAndSettings.TICKET_MAPPINGS;
+import static no.unit.nva.indexing.testutils.MockedJwtProvider.setupMockedCachedJwtProvider;
+import static no.unit.nva.search.ticket.TicketParameter.FROM;
+import static no.unit.nva.search.ticket.TicketParameter.SIZE;
+import static no.unit.nva.search.ticket.TicketType.FILES_APPROVAL_THESIS;
+import static no.unit.nva.search.ticket.TicketType.PUBLISHING_REQUEST;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static nva.commons.apigateway.AccessRight.MANAGE_DEGREE;
+import static nva.commons.apigateway.AccessRight.MANAGE_DOI;
+import static nva.commons.core.attempt.Try.attempt;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import no.unit.nva.commons.json.JsonUtils;
+import no.unit.nva.constants.Words;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.indexingclient.models.EventConsumptionAttributes;
+import no.unit.nva.indexingclient.models.IndexDocument;
+import no.unit.nva.search.common.records.PagedSearch;
+import nva.commons.apigateway.AccessRight;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.apigateway.exceptions.UnauthorizedException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+public class TicketClientTestV2 {
```

Comment:
Creating new test class for tickets to be able to test with single document in index.